libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -253,7 +253,7 @@ open class Node<V : RibView> @VisibleForTesting internal constructor(
         if (isAttachedToView) {
             view?.let { it.attachChild(child, subtreeOf) }
                 ?: parent?.attachChildView(child, this, false)
-                ?: integrationPoint.rootViewHost.attachChild(child, this)
+                ?: integrationPoint.rootViewHost?.attachChild(child, this)
```

Comment:
When `isAttachedToView` is `true`, `rootViewHost` should never be null I guess. So maybe there could be a fallback branch to signal some error?

Reply:
Will add.